resources/dicts/relationship_events/normal_interactions/admiration/decrease.json

```diff
@@ -1,152 +1,215 @@
+			"r_c did an embarrassing prank on m_c's friend."
+		],
+		"reaction_random_cat": {
+			"admiration": "decrease"
+		}
+	},
+	{
+		"id": "admire_de_medium1",
+		"interactions": [
+			"m_c saw r_c take the last piece of prey from the fresh-kill pile.",
+			"m_c thinks r_c has a strange sense of humor.",
+			"m_c is frustrated that r_c won't take {PRONOUN/r_c/poss} actions more seriously.",
+			"m_c feels like r_c isn't taking {PRONOUN/r_c/poss} own personal health seriously.",
+			"m_c has started to see flaws in r_c's philosophies.",
+			"m_c doesn't think r_c does anything productive.",
+			"m_c worries about how capable r_c is.",
+			"m_c doubts r_c would be able to protect {PRONOUN/m_c/object} if something went wrong.",
+			"m_c feels like r_c lied about something trivial.",
+			"r_c told m_c a joke that {PRONOUN/m_c/subject} didn't find funny.",
+			"m_c saw r_c ignore someone.",
+			"m_c wishes r_c would act more maturely.",
+			"r_c said something about a Clanmate that concerned m_c.",
+			"r_c kept getting distracted while helping m_c and nearly got {PRONOUN/m_c/object} hurt."
+		]
+	},
+	{
+		"id": "admire_de_medium2",
+		"interactions": [
+			"m_c tried to bring r_c's behavior up to them, but it turned into a fight that someone else had to break up.",
+			"m_c confronted r_c about {PRONOUN/r_c/poss} bad attitude but that only makes r_c act out more.",
+			"m_c is disappointed when r_c doesn't uphold a promise.",
+			"m_c doesn't tell r_c that {PRONOUN/r_c/subject} forgot to groom a part of {PRONOUN/r_c/poss} fur.",
+			"r_c is visibly upset to be on a patrol with m_c.",
+			"r_c ignored m_c's question.",
+			"In the middle of a conversation, m_c and r_c realize they have less perspectives in common than they thought.",
+			"m_c and r_c talk about a Clanmate and realize they have very different opinions on them.",
+			"r_c tripped m_c. m_c doubts it was an accident.",
+			"m_c scolds r_c for the way {PRONOUN/r_c/subject} acted during a meeting.",
+			"m_c and r_c argue over a piece of prey.",
+			"r_c panics over something that m_c finds trivial."
+		],
+		"reaction_random_cat": {
+			"admiration": "decrease"
+		}
+	},
+	{
+		"id": "admire_de_medium3",
+		"interactions": [
+			"m_c and r_c had to fix something around camp and argued about how to fix it. It ended up being fixed by some other cat in the end.",
+			"m_c notices that r_c has been avoiding {PRONOUN/r_c/poss} responsibilities lately.",
+			"m_c and r_c were supposed to be working together today, but they can never agree long enough to accomplish anything.",
+			"m_c is frustrated after finishing a task that r_c was supposed to do.",
+			"m_c pranked r_c during a patrol.",
+			"m_c is upset that r_c slept through patrol assignments.",
+			"m_c thinks that r_c acted disrespectfully during a patrol.",
+			"m_c noticed that r_c ignored an herb opportunity on a recent patrol.",
+			"After disagreeing on what to do during a recent patrol, m_c is frustrated with r_c."

```

Comment:
You might consider moving some of these to the "admire_adult_de_med1" block.  Many of these aren't appropriate for elders or mediators to take part in.